packages/pcd/webauthn-pcd/test/WebAuthnPCD.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { VerifyAuthenticationResponseOpts } from "@simplewebauthn/server/./dist";
+import { VerifyAuthenticationResponseOpts } from "@simplewebauthn/server";
```

Comment:
This wasn't caught by the lint rule, but I noticed it and saw that it needed fixing.